lib/mpl_toolkits/axes_grid1/inset_locator.py

```diff
@@ -168,24 +168,8 @@ class BboxConnector(Patch):
     @staticmethod
     def get_bbox_edge_pos(bbox, loc):
         """
-        Helper function to obtain the location of a corner of a bbox
-
-        Parameters
-        ----------
-        bbox : `matplotlib.transforms.Bbox`
-
-        loc : {1, 2, 3, 4}
-            Corner of *bbox*. Valid values are::
-
-                'upper right'  : 1,
-                'upper left'   : 2,
-                'lower left'   : 3,
-                'lower right'  : 4
-
-        Returns
-        -------
-        x, y : float
-            Coordinates of the corner specified by *loc*.
+        Return the ``(x, y)`` coordinates of corner *loc* of *bbox*; parameters
+        behave as documented in ``__init__``.
```

Comment:
Use:
```suggestion
        behave as documented in `BboxConnector`.
```
because that creates a link. - Maybe `BboxConnector.__init__` or `.__init__` would work as well, but I've not tested that; and I'm not even sure if that would be better.

Same for all other cases here.

Reply:
done